sonar-text-dotnet/nuget.config

```diff
@@ -1,17 +1,30 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-
-  <packageRestore>
-    <!-- Allow NuGet to download missing packages -->
-    <add key="enabled" value="True" />
-
-    <!-- Automatically check for missing packages during build in Visual Studio -->
-    <add key="automatic" value="True" />
-  </packageRestore>
-
   <packageSources>
     <clear />
-    <add key="slvs_input" value="https://sonarsource.pkgs.visualstudio.com/399fb241-ecc7-4802-8697-dcdd01fbb832/_packaging/slvs_input/nuget/v3/index.json" />
+    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
```

Comment:
We will no longer use the azure feed. The plan is to move away from it.

I've changed this to point to repox and do the package signature validation.